### PR TITLE
SG-13264: Adds .gitattributes

### PR DESCRIPTION
**Pull request by jfboismenu**
On Windows, git doesn't understand pickles and think they are text, so it modifies the line endings. Pickles are actually binary files. This causes the content to be modified and on Python 3 and crashes the parsing of the files, so always have a gitattributes for it.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+# Handle line endings automatically for files detected as text
+# and leave all files detected as binary untouched.
+* text=auto
+
+# Force the following filetypes to have unix eols, so Windows does not break them
+*.pickle text eol=lf
```
Comment (willis102) [inline .gitattributes]:
```suggestion
*.pickle text eol=lf

```

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```
Comment (willis102) [inline .gitattributes]: Does this need to be here?

File: .gitattributes [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# Copyright (c) 2018 Shotgun Software Inc.
```
Comment (willis102) [inline .gitattributes]:
```suggestion
# Copyright (c) 2019 Shotgun Software Inc.
```

File: .gitattributes (hunk 2 above)
Comment (jfboismenu) [reply .gitattributes]: It’s probably there because I copypasted it from a .py file. I’ll remove it

**Pull request merged**